dashboard/settings.py

```diff
@@ -181,14 +181,24 @@
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': False,
+    # Gunicorns logging format https://github.com/benoitc/gunicorn/blob/19.x/gunicorn/glogging.py
+    'formatters': {
+        "generic": {
+            "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
+            "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
+            "class": "logging.Formatter",
+        }
+    },
     'handlers': {
         'console': {
             'class': 'logging.StreamHandler',
+            'formatter': 'generic',
         },
     },
     'loggers': {
         'django': {
             'handlers': ['console'],
+            'propagate': False,
             'level': config('DJANGO_LOG_LEVEL', default='INFO'),
```

Comment:
Currently, the prod logging level is Warning, I would make it INFO at least so that we could watch for errors

Reply:
This is configurable via env variable. I don't think anything needs to be done here. This is just the default.